runtime-watcher/.goreleaser.yml

```
@@ -0,0 +1,60 @@
+# Documentation for how to configure goreleaser can be found at http://goreleaser.com
+project_name: kyma
+release:
+  github:
+    owner: kyma-project
+    name: runtime-watcher
+  prerelease: auto
+before:
+  hooks:
+    - make resolve
+builds:
+  - env:
+      - CGO_ENABLED=0
+    main: ./runtime-watcher/
+    goos:
+      - darwin
+      - linux
+      - windows
+    ignore:
+      - goos: darwin
+        goarch: 386
+archives:
+  - id: foo
+    name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+    format_overrides:
+      - goos: windows
+        format: zip
+checksum:
+  name_template: "checksums.txt"
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  use: github
+  sort: asc
+  groups:
+    - title: Bug fixes
+      regexp: '^.*?fix(\([[:word:]]+\))??!?:.+$'
+      order: 1
+    - title: Dependencies
+      regexp: '^.*?deps(\([[:word:]]+\))??!?:.+$'
+      order: 2
+    - title: Documentation
+      regexp: '^.*?docs(\([[:word:]]+\))??!?:.+$'
+      order: 3
+    - title: Test suites
+      regexp: '^.*?test(\([[:word:]]+\))??!?:.+$'
+      order: 4
+    - title: Features
+      order: 0
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
+      - '^(B|b)ump'
+      - '^.*?chore(\([[:word:]]+\))??!?:.+$'
```

Does not make sense to have a `Documentation` section, but exclude it - Thus remove the exclude statement or remove the docu group. But I would say keep the docu group and remove the exclusion. Same goes for testing.

```suggestion
  groups:
    - title: Bug fixes
      regexp: '^.*?fix(\([[:word:]]+\))??!?:.+$'
      order: 1
    - title: Dependencies
      regexp: '^.*?deps(\([[:word:]]+\))??!?:.+$'
      order: 2
    - title: Documentation
      regexp: '^.*?docs(\([[:word:]]+\))??!?:.+$'
      order: 3
    - title: Test suites
      regexp: '^.*?test(\([[:word:]]+\))??!?:.+$'
      order: 4
    - title: Features
      order: 0
  filters:
    exclude:
      - '^(B|b)ump'
      - '^.*?chore(\([[:word:]]+\))??!?:.+$'
```

---

I took it verbatim from the CLI project. Perhaps we should adjust it there too, then?